src/DataFixtures/ORM/LoadDonationData.php

```diff
@@ -50,13 +55,18 @@ public function load(ObjectManager $manager)
 
         $donation3->stopSubscription();
 
+        $this->setDonateAt($transactionMonthlyError, '-30 day');
         $this->setDonateAt($transaction2, '-1 day');
         $this->setDonateAt($transaction3, '-100 day');
         $this->setDonateAt($transaction4, '-50 day');
 
-        $donator0 = $this->createDonator($donationNormal, '000050', $adherent0);
-        $donator1 = $this->createDonator($donation0, '000051', $adherent1);
-        $donator2 = $this->createDonator($donation4, '000052', $adherent2);
+        $donationMonthly->setLastSuccessfulDonation();
+        $donation0->setLastSuccessfulDonation();
+        $donation3->setLastSuccessfulDonation();
```

Comment:
what do you think to change the method name to `markAsSuccessfulDonation` to avoid any confusion with `Donator->setLastSuccessfulDonation` which takes one arg?